devfiles/apache-camel-springboot-che7.yaml

```diff
@@ -0,0 +1,56 @@
+---
+displayName: Apache Camel based projects on Che 7
```

Comment:
@sleshchenko @slemeur @skabashnyuk I am coming late to the discussion and there may be some details that you have already discussed but I would like to propose a slightly different structure/format for the registry. 

The devfiles in this registry are going to be the reference devfiles for any developer who wants to create her own devfile. She will just copy/paste one of these devfile and customize it. But a couple of things are not *standard* in current structure:

- `displayName`, `description` and `icon` are not attributes of a standard devfile 
- the standard devfile name is `devfile.yaml` 

A possible solution would be to have the following structure for a devfile:

```
devfiles
   | 
    ---  apache-camel-springboot   # No mention of che7 here
              |
              |--- meta.yaml       # Contains the stack details as displayName, icon and descriptin
              ' --- devfile.yaml   # A standard devfile
```

Reply:
makes sense to me.

Just a small question/proposal: can icon field be a relative link to icon file that is placed in `"stack"` directory?

Reply:
One more question related to `meta.yaml` of stack, during converting stacks to devfiles, I removed `tags` and `component` fields:
```
  tags: ["ws.next", "Theia", "Java"]
  components:
  - name: Centos
    version: '7'
  - name: OpenJDK
    version: 1.8.0_181
  - name: NodeJS
    version: 8.12.0
  - name: NPM
    version: 6.4.1
```
Should they be in a `meta.yaml`? In the same format or different?
Note that having `components` field in `meta.yaml` and `devfile.yaml` may be confusing, but for me, it looks clear.

Reply:
Yes I mean for the icon I would make it even simpler: use a standard name (e.g. logo.svg) so that you don't even need to add it to the meta.yaml.

Reply:
And also about `publisher`, for stacks we used to have `publisher` specified.
I think we can just drop it out, but wanted to get confirmation that we really do not need it.
Otherwise, folder structure may be more similar to that we have for Plugin Registry

Reply:
>
>  ```
>  devfiles
>    | 
>     ---  apache-camel-springboot   # No mention of che7 here
>               |
>               |--- meta.yaml       # Contains the stack details as displayName, icon and descriptin
>               ' --- devfile.yaml   # A standard devfile
>  ```

Just wanted to leave here alternative way not to mix `devfile` and `meta` fields. Just the same folder structure that we have now, but object is like the following:
```yaml
displayName: ----
description: 
icon:
devfile:
  projects:
  components:
  ...
```
I agree with Mario that from reusing devfiles (copy/paste) parspective, a separate file with devfile looks more convenient.

Reply:
Let's leave legacy `components` attribute out. You can add `tags` and `publisher` to meta.yaml (or do that in a separate PR).

Reply:
ok for me

Reply:
Changed to 

> ```
> devfiles
>   | 
>    ---  apache-camel-springboot   # No mention of che7 here
>              |
>              |--- meta.yaml       # Contains the stack details as displayName, icon and descriptin
>              ' --- devfile.yaml   # A standard devfile
> ```